packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -159,7 +159,44 @@ export class PipelineService {
     ).then(response => {
       let dialogTitle;
       let dialogBody;
-      if (response['run_url']) {
+      if (response['git_url']) {
+        // pipeline executed in an apache airflow runtime
+        dialogTitle = 'Job submission to ' + runtimeName + ' succeeded';
+        dialogBody = (
+          <p>
+            Apache Airflow DAG has been pushed to the{' '}
+            <a
+              href={response['git_url']}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              GitHub Repository.
+            </a>
+            <br />
```

Comment:
You should just add this chunk of code in the current `dialogBody` wrapped in the `if (response['git_url'])`

Reply:
updated